<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_GetIP</class>
 <widget class="QDialog" name="Frm_GetIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择IP</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>IP列表</string>
   </property>
   <widget class="QListWidget" name="show_IPs_ListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>端口</string>
   </property>
   <widget class="QLineEdit" name="get_TcpIp_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OK_PushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
